api/integrations/github/serializers.py

```diff
@@ -14,6 +17,7 @@ class Meta:
 class GithubRepositorySerializer(ModelSerializer):
     class Meta:
         model = GithubRepository
+        optional_fields = "search_text" "page"
```

Comment:
Looks like a typo? Should this be 
```suggestion
        optional_fields = ("search_text", "page")
```

Reply:
Done